guides/Install-API-instance.md

```diff
@@ -0,0 +1,109 @@
+
+## Install OpenFisca Web API
+
+The following sub-sections describe how to run the `openfisca-api` _playbook_ with 2 scenarios:
+
+- on a remote server for production
+- on a local virtual machine (VM) for development
+
+### Remote server scenario
+
+This scenario describes how to install OpenFisca Web API on a remote server.
+
+There are some prerequisites to running the _playbook_:
+
+- a server must be available, running [Ubuntu 20.04.2.0 LTS (Focal Fossa)](https://releases.ubuntu.com/20.04/)
+- it must expose an SSH connection
+- the _control node_ (the machine that runs the _playbook_) must be able to access the server via SSH as `root`
+- a domain name must have been created, and it must target the public IP address of this server
+
+The _playbook_ refers to the server using `openfisca_api`, a symbolic host name that does not necessarily correspond to a real server host name or domain name (cf `hosts` property in [`openfisca-api.yml`](../ansible/openfisca-api.yml)).
+To associate a concrete IP address or domain name to that symbolic host name, we use an Ansible inventory that can define host groups with a symbolic name (i.e. `openfisca_api` in this case) targeting one or many concrete host names.
+
+Inventories are stored as sub-directories of the [`inventories`](../ansible/inventories/) directory. Each inventory provides at least a `hosts` file, and can also overload _playbook_ default variables by providing a `group_vars/openfisca_api.yml` file.
+
+An SSL certificate is issued from Let's Encrypt if `enable_ssl` and `letsencrypt_email` variables are both defined.
+
+To run the _playbook_, run the following command.
+
+```bash
+ansible-playbook -i ansible/inventories/scaleway/ ansible/openfisca-api.yml
+```
+
+### Local VM scenario
+
+This scenario describes how to install OpenFisca Web API on a virtual machine (VM) managed locally on your development machine by [Vagrant](https://www.vagrantup.com/), a well-known open-source tool.
+
+To install Vagrant, check [its documentation](https://www.vagrantup.com/docs/installation).
+
+Vagrant relies on a [provider](https://www.vagrantup.com/docs/providers) to create a VM. The recommended one is [VirtualBox](https://www.virtualbox.org/), so install it by following [VirtualBox documentation](https://www.virtualbox.org/manual/UserManual.html#installation).
+
+For information, during the redaction of this guide, we used Vagrant version 2.2.16 and VirtualBox 6.1.22.
+
+To create the VM and run the _playbook_, just do:
+
+```bash
+# Ensure you are in the directory containing the Vagrantfile
+cd openfisca-ops
+
+vagrant up
+```
+
+About a minute later, the command will finish, and you should have a VM with OpenFisca Web API running. Thanks to Vagrant port forwarding, the internal port 8000 inside the VM is forwarded to the port 80 of your development machine.
```

Comment:
Let's not assume CPU and internet connection speed and set expectations too high 😉 
If it is the first use of Vagrant on that machine, the initial box download might be pretty long.

```suggestion
You should then have a VM with OpenFisca Web API running. Thanks to Vagrant port forwarding, the internal port 8000 inside the VM is forwarded to the port 80 of your development machine.
```